Frontend/research/src/components/Categories/Art.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { getLinks, deleteLink, updateLink } from "../../Actions/Index";
+
+class Art extends Component {
```

Comment:
doesnt look like you are using state in this file, you could refactor this to a functional component and just use the props values and methods like you are, no need for a class here